assets/javascripts/initializers/spongepowered-navbar.js.es6

```diff
@@ -81,8 +81,8 @@ export default {
         }
       },
 
-      ulTo(linkTo, id, icon, text) {
-        const attributes = { a: { href: linkTo }, li: {} };
+      ulTo(linkTo, id, icon, text, image) {
```

Comment:
I'm not a big fan of this image variable, mostly because

* It's a boolean, which is entirely unclear
* It seems a bit arbitrary that it *also* applies the `ore-logo` class

Maybe change `icon` to take either a string, indicating a FA icon, or a virtual-dom hyperscript element?

Reply:
Sounds good.